### DISCUSSION TITLE
Kindly provide reviews for my GitHub Profile README

### DISCUSSION THREAD

**Discussion by deepansharya1111** [EddieHubCommunity/support#1922] [Q&A] [answered]
Hi everyone 👋 I recently made some changes to my Profile's README file, please provide your views or suggestions on it.
If you like it or if I can improve something? 
The changes I've made are somewhat different from normal profiles so I'm curious to know your views. 
Please ignore the low green squares and contribution count: 😅 I have just started.
Here's my profile link: https://github.com/deepansharya1111

**Comment by schmelto** [comment, marked as answer]
Moved to #1923

**Reply by eddiejaoude** [reply]
Thanks 👍
